dev-python/Kivy/Kivy-1.10.1.ebuild

```diff
@@ -23,7 +23,7 @@ RDEPEND="
 	media-libs/mesa[gles2]
 	dev-python/pillow[${PYTHON_USEDEP}]
 	cairo? ( dev-python/pycairo[${PYTHON_USEDEP}] )
-	camera? ( media-libs/opencv:0/2.4[$(python_gen_usedep 'python2_7')] )
+	camera? ( media-libs/opencv[${PYTHON_USEDEP}] )
```

Comment:
Have you verified that the slot dep is unnecessary? If I see something, I'd expect the package to do something really ugly.

Reply:
I've just removed the whole `camera?` dep